### PR TITLE
New event: CONTENT_DOCUMENT_UNLOADED

### PR DESCRIPTION
**Pull request by JCCR**
While working on #174, I figured it would be nice if this event existed.

It gets triggered whenever a spine item's content doc is unloaded.

Works for all view types.

### CODE REVIEW

File: js/views/reflowable_view.js [hunk 1]
```diff
@@ -176,6 +176,7 @@ ReadiumSDK.Views.ReflowableView = function(options, reader){
 
             //create & append iframe to container frame
             renderIframe();
+            self.trigger(ReadiumSDK.Events.CONTENT_DOCUMENT_UNLOADED, _$iframe, _currentSpineItem );
```
Comment (danielweck) [inline js/views/reflowable_view.js]: Can `_currentSpineItem` be null/undefined here? (in which case the event should not be triggered)

**Comment by danielweck** [comment]
Thanks Juan, this looks great (I have not tested though)
Just one code comment inline (shown above too)

**Comment by JCCR** [comment]
@danielweck You are right. I had that check in one_page_view, looks like I forgot it here.

**Comment by JCCR** [comment]
I found a use for this to improve the highlights plugin so I think its reasonable to merge this in now

**Comment by danielweck** [comment]
Follow-up PR: https://github.com/readium/readium-shared-js/pull/317
(missing scroll view event forward)

**Pull request merged**